### PR TITLE
fix: add more info to  creationtime format

### LINKED ISSUE #10286: Summary Information at resource level can be enhanced
# Summary
Summary Information can be enhanced with more information for more visibility, like the total time of a resource.

# Motivation

While right now we get to see the resource's time elapsed attached as a block on the resource tile/node, it was seen during interaction with end users that they intuitively went to the summary information level to view all the info for that particular resource. We currently have the created_at.  So information like the total time of a resource can be surfaced as well.  
Screen Shot 2022-08-11 at 12 21 53 PM

# Proposal
surface another field like time_existing/time_elapsed at the summary info level for each resource.

####  difficulty: low

### PR DESCRIPTION
**Pull request by ashutosh16**
Signed-off-by: Ashutosh <mail.ashutosh8@gmail.com>

Fixes #10286 

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

**Comment by ashutosh16** [comment]
Screen Shot 2022-09-06 at 3 01 15 PM

File: ui/src/app/applications/components/application-summary/application-summary.tsx [hunk 1]
```diff
@@ -19,10 +19,9 @@ import {Consumer, ContextApis} from '../../../shared/context';
 import * as models from '../../../shared/models';
 import {services} from '../../../shared/services';
 
-import * as moment from 'moment';
-import {ApplicationSyncOptionsField, REPLACE_WARNING} from '../application-sync-options/application-sync-options';
```
Comment (ashutosh16) [inline ui/src/app/applications/components/application-summary/application-summary.tsx:23]: Removed the `REPLACE_WARNING`  as it no longer used in the code.

**Comment by ashutosh16** [comment]
@rbreeze  Would you be able to review the code? Changes are related to UI polish no functional change.

File: ui/src/app/applications/components/utils.tsx [hunk 2]
```diff
@@ -1120,3 +1120,20 @@ export function appQualifiedName(app: appModels.Application): string {
 export function appInstanceName(app: appModels.Application): string {
     return app.metadata.namespace + '_' + app.metadata.name;
 }
+
+export function formatCreationTimestamp(creationTimestamp: any) {
```
Comment (rbreeze) [inline ui/src/app/applications/components/utils.tsx:1124]: Can we avoid using type `any` here?

**Review by rbreeze — DISMISSED** [review]
LGTM after changing type `any`

**Review by crenshaw-dev — APPROVED** [review]
Looks great, thanks @ashutosh16!

**Pull request merged**